### PR TITLE
Some improvement with README.md

### PR DESCRIPTION
**Pull request by supechicken**
### Changes
- Update Discord icon
- Add dark mode support for the Slack icon
- Since the Slack invite link was not working for a while, added a new mailto link that prompt the reader to send a email to crewonslack@gmail.com
- Update copyright

### CODE REVIEW

**Comment by supechicken** [comment]
Seems the `a href` tag doesn’t works with the dark mode detection, finding a way to fix it…

File: README.md [hunk 1]
```diff
@@ -15,12 +32,10 @@ Prerequisites
...
 
-Please be aware of the fact that developer mode is insecure if not properly configured. Setting a password as instructed in the VT-2 login screen is essential. It is also recommended to enable signed boot:
+Please be aware of the fact that developer mode is insecure if not properly configured. Setting a password as instructed in the VT-2 login screen is recommended. It is also recommended to enable signed boot:
 
-```bash
-sudo chromeos-setdevpasswd
-sudo crossystem dev_boot_signed_only=1
-```
+    sudo chromeos-setdevpasswd
+    sudo crossystem dev_boot_signed_only=1
```
Comment (uberhacker) [inline README.md:38]: Better to not indent for easier copy/pasta.

File: README.md [hunk 2]
```diff
@@ -128,15 +143,9 @@ Latest Packages
...
--------------
-<p><em>Discord is not currently syncing messages with Slack</em></p>
-<p><a href="https://join.slack.com/t/chromebrew/shared_invite/enQtNDA2MTQ5ODQ3NDc2LTA0ZmJlMGFmNmZhOTYwNDE3ZDY0NDA0MWI0OTE3MzJkYTQxN2UxMWQ1YTEzOWFmNTliMGM4NDZjYzY2NjczZGE" target="_blank"><img src="/images/slack.png" alt="Slack Invite" />
-<a href="https://discord.gg/QRrzBXN" target="_blank"><img src="/images/discord.png" alt="Discord Invite" /></a></p>
-
 License
 -------
 
-Copyright 2013-2021 Michal Siwek and [all the awesome contributors](https://github.com/skycocker/chromebrew/graphs/contributors).
+Copyright 2013-2022 Michal Siwek and [all the awesome contributors](https://github.com/skycocker/chromebrew/graphs/contributors).
```
Comment (uberhacker) [inline README.md:149]: @skiman6010: This is your responsibility.  Where were you? :)

File: images/discord.svg [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<path d="M59.1895 9.21341C60.3401 9.21341 61.2728 8.35701 61.2728 7.30058C61.2728 6.24416 60.3401 5.38775 59.1895 5.38775C58.039 5.38775 57.1063 6.24416 57.1063 7.30058C57.1063 8.35701 58.039 9.21341 59.1895 9.21341Z" fill="#5865F2"/>
+<path d="M57.1044 10.531C58.3812 11.0939 59.9649 11.1188 61.271 10.531V18.3816H57.1044V10.531Z" fill="#5865F2"/>
+</g>
+<defs>
+<clipPath id="clip0">
+<rect width="124" height="24" fill="white"/>
+</clipPath>
+</defs>
+</svg>
```
Comment (uberhacker) [inline images/discord.svg:18]: Should we be hosting this?  What if it changes and there are legal ramifications?

File: README.md (hunk 2 above)
Comment (skiman6010) [reply README.md:149]: Hey! Someone stole my job!

File: images/discord.svg (hunk 3 above)
Comment (supechicken) [reply images/discord.svg]: Changed to a direct link

File: README.md (hunk 1 above)
Comment (supechicken) [reply README.md:42]: It shows the same thing as backticks (the extra space will not be shown):
```
This is using backticks
```
    This is using indent

**Comment by supechicken** [comment]
> Seems the `a href` tag doesn’t work with the dark mode detection, finding a way to fix it…

Now it works by adding `#gh-dark-mode-only` to both the invite link and image link

File: README.md (hunk 1 above)
Comment (uberhacker) [reply README.md:42]: This is a code block though so it should have backticks.

**Pull request merged**